app/react/Metadata/components/MetadataForm.js

```diff
@@ -50,6 +51,15 @@ export class MetadataForm extends Component {
 
   renderTemplateSelect(templateOptions, template) {
     if (templateOptions.size) {
+      const templateId = template.get('_id');
+      let sortedTemplates = templateOptions
+        .toJS()
+        .filter(defTemp => defTemp.value !== templateId)
```

Comment:
I think that probably even just `temp` is better than `defTemp`. In this expression you are testing against an arbitrary template from the templates list, not against a default template.

Reply:
removed